tensorflow/python/keras/_impl/keras/layers/core_test.py

```diff
@@ -47,6 +47,10 @@ def test_dropout(self):
                   'noise_shape': [3, 1]},
           input_shape=(3, 2))
 
+    with self.test_session():
```

Comment:
Looks good. How about inserting a comment here ? `# Github issue: 14819`

Reply:
Fine . I'd like to do it.